### PR TITLE
UPDATE: added functionality to create tickets with content description

### PR DESCRIPTION
**Pull request by wiwi-websolutions**
Ability to create tickets added.

Tickets can be created as followed:

```php
$teamleader = new Teamleader(config('services.teamleader.api_group'), config('services.teamleader.api_secret'));
$ticket = new Ticket();
$ticket->setClientEmail($event->clientEmail);
$ticket->setClientName($event->clientName);
$ticket->setSubject($event->subject);
$ticket->setContent($html);

$ticket = $teamleader->addTicket($ticket);
```

### CODE REVIEW

File: Tickets/Ticket.php [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+class Ticket
+{
+    /**
+     * @var integer
+     */
+    private $id;
+
+    /**
+     * @var  string
```
Comment (bjorvack) [inline Tickets/Ticket.php:18]: To many spaces after @var

File: Tickets/Ticket.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    private $clientEmail;
+
+    /**
+     * @var  string
+     * subject
+     */
+    private $subject;
+
+    /**
+     * @var  string
```
Comment (bjorvack) [inline Tickets/Ticket.php:36]: idem

**Comment by wiwi-websolutions** [comment]
We copied this line from the `Product.php` and `Invoice.php`, the extra spaces were also present there. We fixed all om them.

**Comment by dreadnip** [comment]
Looks good. Thanks @wiwi-websolutions!

**Comment by dreadnip** [comment]
Released in 1.17.0: https://github.com/sumocoders/Teamleader/releases/tag/1.17.0

**Pull request merged**